### DISCUSSION TITLE
How others do it. (Please collect and discuss treasury solutions of other projects here)

### DISCUSSION THREAD

**Discussion by Phyloiota** [iota-community/Community-Governance#4] [General]
I would suggest to use this as a collection of links to interesting solutions of how other projects handle this. We can learn from those examples and surely dont need to reinvent the wheel completely.
Please drop links to sites or papers and discuss

**Comment by thatlur** [comment]
DOT has quite an interesting way of managing their treasury. Here's a good article on it:
https://medium.com/polkadot-network/a-walkthrough-of-polkadots-governance-486555a056e0

One of the nice things they have implemented for me is the positive turnout quorum biasing (blue line in second graph)
![image](https://user-images.githubusercontent.com/83808392/117423934-27b2bb00-af19-11eb-9306-d2dd5b3602ae.png)

This means that if there is a low voter turnout then a higher percentage of the vote is needed for something to pass. I like this idea as it prevents controversial ideas with low turnout from getting through with just 52% of the vote

**Reply by datahomelab** [reply]
The turnout biasing is a great idea... nice paper.

**Reply by WernerderChamp** [reply]
This looks like a great idea, but there should still be a minimum turnout, especially for proposals that change a lot of stuff.

**Reply by shonuffy** [reply]
Thanks for this, really think the idea of the short term council that is able to veto a vote is important. Especially with the constant conversation about concerns around gaming the system.

**Comment by ricardroyo** [comment]
Here is a great article about how Dash does it: https://richardred.medium.com/observations-of-the-dash-treasury-dao-c94231b2b5c4
It's one of the oldest Treasuries out there, and they managed to run it somewhat successfully for a while.

**Comment by Deep-Sea-888** [comment]
Here is an interesting video of representatives discussing governance structures. The DAO organizations represented are Polkadot, Bancor, and SwissBorg.

https://www.youtube.com/watch?v=yYXZPsL9F-s

Bancor made a statement that was interesting in which they do not simply allow for a greater than 50% majority vote win; but rather, they require a 2/3rd vote requirement which makes it much more difficult for a whale to game it.

DOT made a good statement, that there is a sliding scale. Where there is a quorum number requirement. So, if there is a small quorum represented (ie. someone initiates a vote on New Year's Eve when people are partying or hungover so a very few people are voting), then a higher majority must be required. So an extreme voter turnout may require a 90% majority for something to pass.

**Reply by basvansambeek** [reply]
I think there should also be a reasonable voting window, maybe connected to the impact of the vote. Small amounts of tokens would then require maybe a week, big investments would have the vote open for something like a month. I don't its reasonable for everyone to be constantly involved in governance.

Speed should not be more important than diligence here, and together with quorum biasing, voting rounds and putting up a stake we could probably find a mix that reduces spam and attack vectors a lot, without hindering genuine proposals much.

**Reply by ricardroyo** [reply]
I like the idea of variable quorum / majority requirements depending on the situation. It will be difficult to estimate what are the right parameters until the DAO would be up and running but if, technology willing, we can adjust these parameters as part of a governance vote, it would be a useful tool IMO. We might have some parameters that work at the beginning when people are very active but lead us to a complete paralysis when excitement disappears. 

What do you guys think about this?:

Quorum could depend on: 
- Proposal amount (ie big proposal - big quorum)

 
Majority could depend on: 
 - Quorum reached on a particular vote 
 - Type of vote (I understand some votes might not be to approve a proposal but to change parameters of the DAO, those could require qualified majority)

I would love to hear your opinions about this

**Comment by Deep-Sea-888** [comment]
In discussing in Discord someone brought up that the blog post stated, "Anyone can enter a proposal".

I look at this upcoming vote as the "initial vote". Such that the governance of the initial vote may be less stringent than say a governance vote within the DAO treasury later on.

Hearing that anyone can submit a proposal how does the initial vote protect from a proposal such as: "Give all 65 Ti to ThomasQV"? And then, ThomasQV being the ultimate whale he may be, he games the vote.

Focusing on the proposal BAS had discussed tiered voting rounds. This could work for the proposals. Such that maybe it's 3 rounds or 4 rounds to get down to 4, 3, or even 2 choices. 

The initial set of proposals may have spam-like, "give all the 65 Ti to the Ethereum Foundation or fund everyone who was active on SPEC on May 1st, 2021 to go to Moonaco". With a tiered / round system everyone initially sees the spam. If any of the ridiculous proposals get approved and move to the next round, it becomes clear that there may be gaming in the system.

Another thought as well is some sort of fee or stacking for proposal submittals?

**Reply by WernerderChamp** [reply]
We had a discussion about this on monday. Some other project (was it DOT?) has a 5% fee on a proposal. So if you want to take 10 Gi out of the DAO, you would have to deposit 500 Mi first. If the proposal is rejected, the funds are burned.

We discussed about having two no votes, one where the funds are burned, one where the funds (or part of it) are sent back. In case a proposal is rejected, the funds will only be burned if the "reject and burn" votes have a majority.

**Reply by shonuffy** [reply]
Also, I don't think the Treasury would be set up to distribute the funds in a one big payment. It would likely be separated into sub-DAOs with different caps, for example $1,000, $5,000, $50,000 etc.

**Comment by Phyloiota** [comment]
Someone did make me aware of this website:  https://snapshot.org/#/ - think that' s an amazing tool to see how others do it... a big overview of all the current ongoing votings in different projects....
The new YFI governance proposal stood directly out for me... pretty well developed imho so far...
https://gov.yearn.finance/t/yip-61-governance-2-0/10460

**Comment by sascha1337** [comment]
https://commonwealth.im/edgeware/proposal/discussion/831-decent-an-edgeware-ecosystem-project-update-150321
https://wiki.polkadot.network/docs/en/learn-treasury
https://www.decent.partners/
https://edgewa.re/

**Comment by Phyloiota** [comment]
Putting this up here as i found it useful and informative. Its a good summary of the Governance structure used in Polkadot. Not all those principles seem to be useful or even needed for the IOTA Treasury, but they have definetly put a big amount of research into the topic.

https://medium.com/coinmonks/understanding-polkadots-governance-mechanism-6-key-questions-and-answers-1245b91f9e43

And here is the detailed description of the phragmen algorythm that is used to rotate DOT council members atm:

https://wiki.polkadot.network/docs/en/learn-phragmen

and the planned updated and changed version of it to be implemented:

https://arxiv.org/pdf/2004.12990.pdf

**Reply by Phyloiota** [reply]
I think it is definetly worth to have a deep look into this system and take what we find useful.

**Reply by basvansambeek** [reply]
It’s quite a read and I notice that there are a lot of game theoretical assumptions where the implications are not fully expanded on.

For instance the conviction factor basically favors wealthy holders, making the system de facto super linear, because they have more tokens that can easily be staked for longer. An immediate question is if this is an intended outcome, or is it an externality? 

The article mentions the ambition of being as permissionless as possible, but that a hard to define quality, and even harder to quantify in reality. It’s certainly a challenge to implement in a fair way, given how soon it becomes very complex.

I’d love to know how they dealt with scope and complexity, after we’ve established what values an IOTA DAO should adhere to.

**Reply by Phyloiota** [reply]
yepp - same thoughts i have ... intentionally it might sound good to give a longer stake more power, but it will absolutely not help to make "small holders" more powerful... Someone with large holdings and a strong agenda towards a decision gains kind of superpower with this... so it is kind of failing in what they seems to plan to establish with it...
Interesting, but very complex is the council rotation based on the phlegma algo... but yeah quiet a read ...
They have also definetly a different focus, as it is mainly a voting on protocol changes - that have the potential to damage the code etc... so that not our concern and out of the scope of IOTA DAO luckily...
I think the biased quorum balancing can be a useful tool if crafted carefully to avoid this "low interest" proposals captured by a small majority of their own supporters...

**Reply by basvansambeek** [reply]
Out of curiosity: did you misspell Phragrém on purpose as a critique? Or is it autocorrect?

**Reply by Phyloiota** [reply]
> Out of curiosity: did you misspell Phragrém on purpose as a critique? Or is it autocorrect?

Haha, lol - no did not recognize that... haha ... seems to be a "Freudscher Versprecher" then...but with no bad intentions haha. Thanks for the hint... ill leave it misspelled like that, otherwise this little chat looses its sense....Phlegma .... lol

**Reply by basvansambeek** [reply]
It seems the Medium post also has a misspelling. Phragmén gives more results, for those googling it.

**Comment by Phyloiota** [comment]
I did some research in the last weeks and dived a bit in the crypto governance systems that are already out there. Surely the IOTA community does not need to invent everything new in this huge topic, and we can learn from others about success and and problematics of community governance. But clearly we will need to find our own solution.
I did put my findings a summary together in this document. I dont claim this doc of being perfect, its just what everyone else that uses google can find out about the state of community governance in Crypto. The doc is open for commenting and i would be happy if you use the function in the doc to put up suggestions or hint me if i did post wrong information here.
https://docs.google.com/document/d/1JGuPFQNooMSZhHujEgOTZFwRjcRPeTMCDtlEmrsWF7g/edit?usp=sharing
